app/util/conf/base_config_loader.py

```diff
@@ -135,6 +137,20 @@ def load_from_config_file(self, config, config_filename):
         if self.CONFIG_FILE_SECTION:
             self._load_section_from_config_file(config, config_filename, self.CONFIG_FILE_SECTION)
 
+    def _override_cacerts(self, root_directory):
```

Comment:
Do you think it might be better to add this logic in main.py instead? It's not really related to config loading, so I'm not sure it belongs in BaseConfigLoader.

Reply:
How about putting it into `BaseConfigLoader.configure_postload` section?
This change is only required if we are using HTTPS protocol.

Reply:
Here 
```
        if Configuration['https_cert_file'] and Configuration['https_key_file']:
            conf.set('protocol_scheme', 'https')
            _override_cacerts(root_directory)
```

Reply:
That sounds good.
\